func/domain.sh

```diff
@@ -719,6 +726,7 @@ del_mail_ssl_config() {
 
     # Remove SSL certificates
     rm -f $HOMEDIR/$user/conf/mail/$domain/ssl/*
+    rm -f /usr/local/hestia/ssl/mail/$domain.*
```

Comment:
Same but then the delete and the other way round 

General note:
We should delete .crt and .key separate to prevent issues with domain.com.au and domain.com and then deleting domain.com